src/cluster_legacy.c

```diff
@@ -169,7 +169,7 @@ dictType clusterSdsToListType = {
         NULL                        /* allow to expand */
 };
 
-/* Aux fields are introduced in Redis 7.2 to support the persistence
+/* Aux fields were introduced to support the persistence
```

Comment:
```suggestion
/* Aux fields were introduced in Redis OSS 7.2 to support the persistence
```

I think it can be good to keep this, since it's fairly new. In a cluster, nodes need to communicate with nodes running different versions.